gobblin-core/src/main/java/gobblin/source/extractor/extract/jdbc/JdbcExtractor.java

```diff
@@ -12,6 +12,27 @@
 
 package gobblin.source.extractor.extract.jdbc;
 
+import gobblin.configuration.ConfigurationKeys;
```

Comment:
Can you revert back the import order (https://github.com/linkedin/gobblin/wiki/CodingStyle)?